servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -0,0 +1,146 @@
+[[threescale-adapter-tutorial]]
+= Appendix: 3scale Istio Adapter tutorial
```

Comment:
We could probably remove the word "Appendix" here.  And the word "tutorial" since this isn't really a tutorial.

[[threescale-adapter]]
= 3scale Istio Adapter